test-cases/longevity/longevity-alternator-200GB-48h.yaml

```diff
@@ -0,0 +1,33 @@
+test_duration: 180
+prepare_write_cmd: [
+  "bin/ycsb load dynamodb -P workloads/workloada -threads 250 -p recordcount=200200300 -p fieldcount=1 -p fieldlength=1024 -p dataintegrity=true -p hdrhistogram.summary.addstatus=true", ]
```

Comment:
I understand you copied it from the current c-s but I don't know if it make sense.
You need to consult the team about a typical schema of dynamo or google it.
I don't think that fieldcount=1 and fieldlength=1024 is something that will help us to catch real life bugs.

Reply:
Also, why do we need a prepare?
Is this stage a write only?

Reply:
Yes this is write only, as for sizes, we just know dynamodb has 400K limit per request